assets/training/finetune_acft_hf_nlp/environments/acpt-draft/context/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN pip install transformers==4.57.1
 RUN rm -rf ~/.cache/pip
 RUN rm -rf /opt/conda/lib/python3.10/site-packages/pip-25.2.dist-info
 RUN rm -rf opt/conda/lib/python3.13/site-packages/pip-25.2.dist-info
+RUN rm /opt/conda/envs/ptca/conda-meta/pip-25.2-pyhc872135_0.json
```

Comment:
better to not use pip but conda to update pip. It resolved the issue and update json correctly

Reply:
example => https://github.com/Azure/azureml-assets/pull/4653